### ISSUE TITLE
timestamp_t not creating datetime counterpart in object

### ISSUE THREAD

**Issue by wjdingman1** [ocsf/ocsf-server#39]
Introducing a custom timestamp_t attribute in extension creates the counterpart in the dictionary, but not in the object.

Steps to reproduce:

1. Create new `timestamp_t` attribute in an extension dictionary, add it as field in object.
2. Run server and navigate to object
3. Toggle date/time profile, datetime_t attribute does not appear

**Comment by rroupski-splunk** [comment]
@wjdingman1 can provide the attribute snippets that you used in the dictionary and the object ?

**Comment by wjdingman1** [comment]
```json
 "password_expiry_time": {
      "caption": "Password Expiry Time",
      "description": "The time when the password expires. See specific usage.",
      "type": "timestamp_t"
    }
```

```json
  "password_expiry_time": {
      "description": "The time when the user's password will expire.",
      "requirement": "optional"
    }
```

**Comment by rroupski** [comment]
Fixed in commit #[97cf9a8a](https://github.com/ocsf/ocsf-server/commit/97cf9a8a25db4a3f3646fbe400123f91c73a59eb)
